src/workspace-hack/Cargo.toml

```diff
@@ -172,7 +172,8 @@ serde_json = { version = "1", features = ["alloc"] }
 serde_with = { version = "2", features = ["json"] }
 smallvec = { version = "1", default-features = false, features = ["serde"] }
 subtle = { version = "2" }
-syn = { version = "1", features = ["extra-traits", "full", "visit", "visit-mut"] }
+syn-dff4ba8e3ae991db = { package = "syn", version = "1", features = ["extra-traits", "full", "visit", "visit-mut"] }
+syn-f595c2ba2a3f28df = { package = "syn", version = "2", features = ["full", "visit-mut"] }
```

Comment:
Why is this needed? Seems syn is not used in this PR.

Reply:
May related to OpenDAL version `0.33.1`? I just bump version.

Reply:
Can we ban `typo checker / Spell Check with Typos (pull_request) ` for .toml?